fullcalendar-app/src/components/FullCalendar.vue

```diff
@@ -504,4 +506,7 @@ export default {
 .fc-event-time {
   opacity: 80%;
 }
+.fc-day-today {
+  background-color: inherit !important;
```

Comment:
Why do we need `!important` here?

Reply:
We don't need this

Reply:
Let's remove then. Less `!important` - more predictable UI